helpers/TaskQueue.js

```diff
@@ -104,6 +126,8 @@ class TaskQueue {
       console.log('Encountered an error while dequeuing the next task:', err);
       return;
     } finally {
+      // Update dequeue timestamp
+      this.timeOfLastDequeue = new Date().getTime();
```

Comment:
Can we use `Date.now()` instead?

Reply:
Whoops, you're right - I was under the impression that `Date.now()` is equivalent to `new Date()`, but it's actually equivalent to this. Will switch.